helix-core/src/main/java/org/apache/helix/controller/stages/CustomizedStateComputationStage.java

```diff
@@ -86,6 +86,9 @@ private void updateCustomizedStates(String instanceName, String customizedStateT
           customizedStateOutput
               .setCustomizedState(customizedStateType, resourceName, partition, instanceName,
                   customizedState.getState(partitionName));
+          customizedStateOutput
```

Comment:
I added another parameter to setCustomizedState() and now this method can set both fields.
Originally I didn't want to change the interface of setCustomizedState() because it is a public method. But I think your comment also makes sense since we are going to keep this start time field, and this is the only place setCustomizedState()  is called.